Factory/DiactorosFactory.php

```diff
@@ -49,7 +50,14 @@ public function createRequest(Request $symfonyRequest)
             $body = new DiactorosStream('php://temp', 'wb+');
             $body->write($symfonyRequest->getContent());
         } else {
-            $body = new DiactorosStream($symfonyRequest->getContent(true));
+            $bodyStream = $symfonyRequest->getContent(true);
+            $bodyStreamMetaData = stream_get_meta_data($bodyStream);
+
+            if($bodyStreamMetaData['uri'] == 'php://input') {
```

Comment:
We always use Yoda-style conditions and strict comparison (`'php://input' === $bodyStreamMetaData['uri']`).